pkg/broker/filter/filter_handler_test.go

```diff
@@ -246,7 +246,7 @@ func TestReceiver(t *testing.T) {
 				makeTrigger(makeTriggerFilterWithAttributesAndExtension(eventType, eventSource, "some-other-extension-value")),
 			},
 			event:              makeEventWithExtension(extensionName, extensionValue),
-			expectedEventCount: true,
+			expectedEventCount: false,
```

Comment:
I'm curious why this change impacted the expectedEventCount to not be counted? I'd expect (from the test name) that the filter passed, but delivery to destination failed? If so, why wouldn't it bump the event count?

Reply:
Good catch :) The name of the test was wrong. It didn't even tried to dispatch, as the filter doesn't match because of different extensions. Just updated the name of the test.